main/guides/getting-started/start-a-project.md

```diff
@@ -3,6 +3,16 @@
 After you've [installed the Agoric SDK](./README.md) (_recall: use `agoric --version` to confirm._), you're ready for your first _Agoric Dapp_ (decentralized application).
 
 
+::: tip Beta Dapp Architecture
```

Comment:
I am not familiar with this repo, is there an `::: alert` or `::: info`we can use that's maybe more prominent? Definitely noticeable, but wondering if we can make it stand out a little more.